src/fn/compile-app-settings.js

```diff
@@ -7,11 +7,12 @@ const compileNoolsRules = require('../lib/compile-nools-rules');
 const fs = require('../lib/sync-fs');
```

Comment:
Would you write release notes for this in `release-notes.md`?

Reply:
I added a note in release notes.